plans/tasks.py

```diff
@@ -4,11 +4,33 @@
 from celery.task.base import periodic_task
 from django.conf import settings
 from django.contrib.auth import get_user_model
+from .signals import account_automatic_renewal
 
 
 User = get_user_model()
 logger = logging.getLogger('plans.tasks')
 
+
+@periodic_task(run_every=crontab(hour=0, minute=5))
+def autorenew_account():
+    logger.info('Started')
```

Comment:
The logger could be more informative, at least containing the task name or what it's going to happen.